website/addons/owncloud/serializer.py

```diff
@@ -0,0 +1,81 @@
+from website.addons.base.serializer import OAuthAddonSerializer
+from website.addons.owncloud.settings import DEFAULT_HOSTS
+from website.util import api_url_for, web_url_for
+from website.addons.owncloud.utils import ExternalAccountConverter
+
+class OwnCloudSerializer(OAuthAddonSerializer):
+
+    addon_short_name = 'owncloud'
+
+    REQUIRED_URLS = []
+
+    # Include host information with more informative labels / formatting
+    def serialize_account(self, external_account):
+        ret = super(OwnCloudSerializer, self).serialize_account(external_account)
+        converted = ExternalAccountConverter(external_account)
+        ret.update({
+            'host': converted.host,
+            'host_url': 'https://{0}'.format(converted.host),
+        })
+
+        return ret
+
+    def serialized_folder(self, node_settings):
+        return {
+            'name': node_settings.folder_name,
+            'path': node_settings.folder_name
+        }
+
+    @property
+    def credentials_owner(self):
+        return self.node_settings.user_settings.owner
+
+    @property
+    def serialized_urls(self):
```

Comment:
If you're using `StorageAddonSerializer` and don't unset `REQUIRED_URLS`, the `super` implementation of this should be fine.